### PR TITLE
Linux bignum fix

### LINKED ISSUE #41: Build fails on Linux : use of undeclared type 'BIGNUM'
Building on Linux fails with several of the following

```
/srv/datadisk01/www/VideoSaverServer/.build/checkouts/APNSwift/Sources/APNSwift/APNSwiftSigner.swift:56:31: error: use of undeclared type 'BIGNUM'
        var r : UnsafePointer<BIGNUM>? = nil
```

Swift version 5.1-dev
Ubuntu 18

### PR DESCRIPTION
**Pull request by kylebrowning**
Attempts to fix #41

### CODE REVIEW

File: Sources/APNSwift/APNSwiftSigner.swift [hunk 1]
```diff
@@ -40,29 +40,34 @@ public struct APNSwiftSigner {
...
-        var r : UnsafePointer<BIGNUM>? = nil
-        var s : UnsafePointer<BIGNUM>? = nil
+        var r : OpaquePointer? = nil
+        var s : OpaquePointer? = nil
         
         // as this method is `get0` there is no requirement to free those pointers: ECDSA_SIG will free them for us.
-        CAPNSOpenSSL_ECDSA_SIG_get0(.init(sig), &r, &s)
+        withUnsafeMutablePointer(to: &r) { rPtr in
+            withUnsafeMutablePointer(to: &s) { sPtr in
+                CAPNSOpenSSL_ECDSA_SIG_get0(.init(sig), .make(optional: rPtr), .make(optional: sPtr))
```
Comment (weissi) [inline Sources/APNSwift/APNSwiftSigner.swift]: I think you can drop both `.make(optional: ...)` here, just `rPtr` and `sPtr` should work

**Comment by weissi** [comment]
CC @lukasa

File: Sources/APNSwift/APNSwiftSigner.swift (hunk 1 above)
Comment (kylebrowning) [reply Sources/APNSwift/APNSwiftSigner.swift]: When I did that I get the following.

Cannot convert value of type 'UnsafeMutablePointer<_>' to expected argument type 'UnsafeMutablePointer<UnsafePointer<BIGNUM>?>?' (aka 'Optional<UnsafeMutablePointer<Optional<UnsafePointer<bignum_st>>>>')

Comment (weissi) [reply Sources/APNSwift/APNSwiftSigner.swift]: oh yes, ignore me.

**Pull request merged**